whois-client/src/main/java/net/ripe/db/whois/api/rest/domain/AbuseContact.java

```diff
@@ -19,14 +19,24 @@
     @XmlAttribute(name = "email")
     private String email;
 
-    public AbuseContact(final String key, final String email) {
+    @XmlAttribute(name = "suspect")
+    private boolean suspect;
+
+    @XmlAttribute(name = "org-id")
```

Comment:
Do we need to return the sponsoring org-id separately from the org-id ?

Reply:
It's the org-id of the org to contact about the abuse-c, we could have seperate entries but then the caller still needs to figure out which one to use.

Reply:
As discussed, if it's clear the different use of this org-id specifically for the abuse contact, then there's no need to return separately (the backend will return the appropriate one).